src/NUnitFramework/tests/Constraints/EmptyConstraintTest.cs

```diff
@@ -123,5 +123,16 @@ public void NotEmptyDirectory()
                 Assert.That(testDir.Directory, Is.Not.Empty);
             }
         }
+
+        [Test]
+        public void NotEmptyDirectory_ContainsDirectory()
```

Comment:
Added because while there was a test for a directory containing files, there wasn't for when the directory only contains subdirectories